samples/scholarly/scholarly.css

```diff
@@ -0,0 +1,496 @@
+}
+section:target h2, section:target h3, article:target h1 {
+  color: #2196f3;
+}
+@media screen {
+  article {
+    /* the padding is 116px 116px 80px 116px, from which we remove the border */
+    padding: 88px 88px 72px 108px;
+    background: #fff;
+    border-top: 28px solid transparent;
+    border-right: 28px solid transparent;
+    border-bottom: 8px solid transparent;
+    border-left: 4px solid transparent;
+    border-image: url('card-bg.png') 28 28 8 4 repeat;
+  }
+}
+article {
+  font-size: 13px;
+  font-weight: 300;
+  line-height: 24px;
+}
+p {
+  text-align: justify;
+  -moz-hyphens: auto;
+  hyphens: auto;
+  widows: 2;
+  orphans: 2;
+  margin-top:0;
+  margin-bottom: 1em;
+}
+h1, h2, h3 {
+  font-variant: small-caps;
+  text-align: center;
+  font-weight: 300;
+  break-after: avoid;
+  margin-bottom: 1em;
+}
+h1 {
+  padding-top:  16px;
+  padding-bottom: 16px;
+  letter-spacing: .75px;
+  font-size: 24px;
+  line-height: 32px;
+}
+h2 {
+  padding-top: 16px;
+  padding-bottom: 0;
+  letter-spacing: .75px;
+  font-size: 20px;
+}
+article *:first-child {
+  padding-top: 0;
+  margin-top: 0;
+}
+figure:first-child {
+  padding-top: 0 !important;
+  margin-top: 0 !important;
+}
+h3 {
+  padding-top: 16px;
+  padding-bottom: 0;
+  letter-spacing:.5px;
+  font-size: 16px;
+}
+dfn {
+  text-decoration: underline;
+}
+a {
+  color: #2196f3;
+}
+a, :link, :visited {
+  text-decoration: none;
+  color: #2196f3;
+}
+a:active {
+  color: #64b5f6;
+}
+figure img {
+  max-width: 100%;
+  width: auto;
+  height: auto;
+}
+
+pre, code, a[property="schema:citation"] {
+  -moz-hyphens: none;
+  hyphens: none;
+}
+code {
+  color: #FF8C00;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  border-radius: 0.3em;
+  padding: 0.2em;
+  background: #FDFDFD;
+  display: inline;
+}
+code.language-html {
+  color: inherit;
+}
+
+pre code {
+  color: inherit;
+  border: none;
+  padding: 0;
+  background: none;
+}
+
+footer {
+  font-size: 13px;
+  box-shadow: -1px 0px 2px #BCBCBC;
+  background: #BDD5E9;
+  /*border-top: 2px solid #F9EABD;*/
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+}
+footer p {
+  max-width: 900px;
+  padding: 14px 10px;
+  margin: 0 auto;
+  text-align: center;
+}
+footer a:link, footer a:visited {
+  color: white;
+}
+footer img {
+  vertical-align: -4px;
+  background: white;
+  border-radius: 3px;
+  padding: 1px;
+  box-sizing: content-box;
+}
+
+section {
+  margin-bottom: 6em;
+}
+section[typeof='sa:ReferenceList'] dt {
+  float: left;
+  margin-right: 5px;
+}
+section[typeof='sa:ReferenceList'] dt::after {
+  content: ',';
+}
+section[typeof='sa:ReferenceList'] dd {
+  margin-bottom: 16px;
+}
+section[typeof='sa:ReferenceList'] cite::before {
+  content: '«';
+}
+section[typeof='sa:ReferenceList'] cite::after {
+  content: '»';
+}
+
+code {
+  overflow: visible;
+  white-space: pre-wrap;
+}
+pre {
+  border: solid 1px #DFDFDF;
+  border-left: solid #358CCB 10px;
+  max-width: 100%;
+  box-sizing: border-box;
+  margin:0;
```

Comment:
You mean in the pre { } block? yes, probably, although not too much as we have fairly long lines and not very wide boxes for pre, so I'd prefer not to wrap lines too much.

How about 1em?